shared/uavobjectdefinition/manualcontrolsettings.xml

```diff
@@ -105,13 +105,15 @@
 				<option>Always Armed</option>
 				<option>Roll Left</option>
 				<option>Roll Right</option>
-				<option>Pitch Forward</option>
-				<option>Pitch Aft</option>
 				<option>Yaw Left</option>
 				<option>Yaw Right</option>
+				<option>Corners</option>
 				<option>Switch</option>
+				<option>Switch+Throttle</option>
 			</options>
 		</field>
+		<field name="ArmTime" units="ms" type="enum" elements="1" options="250,500,1000,2000" defaultvalue="1000"/>
+		<field name="DisarmTime" units="ms" type="enum" elements="1" options="250,500,1000,2000" defaultvalue="1000"/>
```

Comment:
I think we should have longer disarm timeouts. Someone might want to trun off throttle on a multirotor for a while, so it falls for a while and then throttle up again. It woud be bad if it disarms in this case. Before it was possible to set the timout in seconds, which wa better IMO.